src/index.ts

```diff
@@ -48,28 +54,26 @@ class LinkSDK {
     };
   }
 
-  // Set the token from callback or server
-  setToken(appToken: string): void {
-    this.params.access_token = appToken;
-  }
-
   // Open My Account to authorize application to use MtLink API
-  authorize(newTab: boolean = false): void {
+  authorize(options: LinkOptions = {}): void {
```

Comment:
Adding a backTo param to /oauth/authorize could be overkill here. Since we already have the the redirect_uri for use on that view it can also be used when the Guest presses cancel.

Reply:
Yeah, I wasn't 100% sure myself.
This comes from the idea that one could come to `/oauth/authorize` from the Vault as you suggested in one of your early flow.

Reply:
For now if we avoid changing parameter on the OAuth page as little as possible, for safety and stability sake, we can always revisit it later if we discover a good use-case via client feedback.